### ISSUE TITLE
Feature Request: Notifications

### ISSUE THREAD

**Issue by m160** [cabal-club/cabal-cli#124]
It would be nice to have the ability to get notifications from cabals/channels/mentions/etc if yous so choose. A sound and pop-up for when you're on another workplace, or if you're awaiting someone to get back to you etc.

Keep up the good work!

**Comment by cblgh** [comment]
nick found this that we could use https://github.com/mikaelbr/node-notifier, i'd implement it as such that it would be off by default. would have to think about managing settings too (but i guess that would be part of the config? we'll seee)

thanks @m160!

**Comment by noffle** [comment]
The way I've seen other command line programs do system notifications is by emitting a `BEEP` ANSI escape sequence. Most terminal emulators will turn this into some kind of visual cue on the window itself.

**Comment by cblgh** [comment]
:o

**Comment by jolyjf** [comment]
Yes please implement this :) Something equivalent to `tput bel` when there is a new unread message... Having the option to enable notification for certain channels only would be nice too.
